verify-push-backend/tests/create-challenge.test.js

```diff
@@ -33,13 +34,23 @@ describe('verify-push-backend/create-challenge', () => {
       '/missing-params.js',
       '../assets/missing-params.private.js'
     );
+    runtime._addAsset(
+      '/digest-message.js',
+      '../assets/digest-message.private.js'
+    );
     helpers.setup(testContext, runtime);
     jest.mock('../assets/missing-params.private.js', () => {
       const missing = jest.requireActual('../assets/missing-params.private.js');
       return {
         detectMissingParams: missing.detectMissingParams,
       };
     });
+    jest.mock('../assets/digest-message.private.js', () => {
+      const missing = jest.requireActual('../assets/digest-message.private.js');
+      return {
+        digestMessage: missing.digestMessage,
+      };
+    });
```

Comment:
Can we add a test that checks that the new hidden details are being processed as expected?

Reply:
Done